Dockerfile.v4.2.x

```diff
@@ -1,18 +1,24 @@
 FROM ubuntu:18.04
 LABEL maintainer="Matt Bentley <mbentley@mbentley.net>"
 
+ARG DEBIAN_FRONTEND=noninteractive
+ARG OMADA_VER=4.2.8
+ARG OMADA_TAR="Omada_SDN_Controller_v${OMADA_VER}_linux_x64.tar.gz"
+ARG OMADA_URL="https://static.tp-link.com/2020/202012/20201211/${OMADA_TAR}"
+
+# download installation tarfile
+ADD ${OMADA_URL} /tmp/${OMADA_TAR}
```

Comment:
Using `ADD` here is a poor practice.  Files that are added can not be fully removed from the image so it inflates the image.  `test1` was the previous version, `test2` is the new version:

```
mbentley/omada-controller   4.2-test2   355e47a1be6a   6 seconds ago        422MB
mbentley/omada-controller   4.2-test1   2642010fdfef   About a minute ago   361MB
```

I would remove this `ADD` line and bring back the manual download and extraction to prevent this.

Reply:
fixed